src/main/java/no/nav/veilarboppfolging/client/veilarbarena/VeilarbarenaClientImpl.java

```diff
@@ -55,10 +57,13 @@ private String getToken() {
 
     @Override
     public Optional<VeilarbArenaOppfolging> hentOppfolgingsbruker(Fnr fnr) {
+    	PersonRequest personRequest = new PersonRequest();
+    	personRequest.setFnr(fnr);
         Request request = new Request.Builder()
-                .url(joinPaths(veilarbarenaUrl, "/api/oppfolgingsbruker/" + fnr.get()))
+                .url(joinPaths(veilarbarenaUrl, "/api/v2/oppfolgingsbruker/"))
```

Comment:
Fjerne trailing-slash:
```suggestion
                .url(joinPaths(veilarbarenaUrl, "/api/v2/oppfolgingsbruker"))
```